src/circuit/operator/inspect.rs

```diff
@@ -47,3 +47,32 @@ where
         (self.callback)(i)
     }
 }
+
+// Sink operator that prints all elements in its input stream.
```

Comment:
I really hate to maintain all these trivial operators.  This should be just `stream.inspect(|x| println!("new input {}", x))`.